src/quo2/components/notifications/toast/view.cljs

```diff
@@ -57,7 +56,7 @@
          :style               (style/text override-theme)
          :accessibility-label :toast-content}
         text])]
-    (when right right)]])
+    right]])
```

Comment:
`(when right right)` is the same as `right` in this case

Reply:
right!